src/sage/schemes/elliptic_curves/ell_field.py

```diff
@@ -1588,6 +1588,62 @@ def isogenies_prime_degree(self, l=None, max_l=31):
         from .isogeny_small_degree import isogenies_prime_degree
         return sum([isogenies_prime_degree(self, d) for d in L], [])
 
+    def isogenies_degree(self, n):
+        """
```

Comment:
```suggestion
        r"""
```
Unnecessary in this case, but inevitably someone will start using backslashes in the docstring at some point in the future, which would then cause trouble.